tests/test_awkward.py

```diff
@@ -13,3 +13,7 @@ def test_to_awkward():
     arr = pylhe.to_awkward(pylhe.read_lhe_with_attributes(TEST_FILE))
     assert len(arr) == 791
     assert len(arr.eventinfo.nparticles) == len(arr.particles)
+    # iss 194 pr 195 - make sure vector helper classes funciton properly
+    #    here we make sure the 'mass' helper function on 4D vectors
+    #    is called on each particle's momentum
```

Comment:
```suggestion
```

This commentary is good for the PR body, but isn't really helpful in the tests. If test aren't clear what they're doing then they should get a docstring, not comments.